plone/recipe/codeanalysis/__init__.py

```diff
@@ -198,6 +199,11 @@ def uninstall_pre_commit_hook(self):
 def code_analysis(options):
     start = time()
 
+    # if there is a second argument (first is always the progam itself)
```

Comment:
I was just wondering because I use argparse all the time for command line arguments and I quite like it. I'm ok with keeping it as it is though if you prefer it that way...